.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,61 @@
+name: 🐞 Bug Report
+description: Report a bug
+labels:
+    - kind/bug
+    - status/needs-triage
+
+body:
+    - type: markdown
+      attributes:
+          value: |
+              ## First of all
+              1. Please search for [existing issues](https://github.com/libgdx/libgdx/issues?q=is%3Aissue) about this problem first.
+              2. Remember to be friendly.
+
+    - type: textarea
+      id: description
+      attributes:
+          label: Describe the bug
+          description: A clear description of what the bug is. Include screenshots if applicable.
+          placeholder: Bug description
+      validations:
+          required: true
+
+    - type: textarea
+      id: reproduction
+      attributes:
+          label: Reproduction
+          description: Steps to reproduce the behavior.
+          placeholder: |
+              1. Go to ...
+              2. Click on ...
+              3. See error
+
+    - type: textarea
+      id: expected-behavior
+      attributes:
+          label: Expected behavior
+          description: A clear description of what you expected to happen.
+
+    - type: textarea
+      id: logs
+      attributes:
+          label: Stack trace
+          render: shell
+    - type: checkboxes
+      id: affected_platforms
+      attributes:
+          label: "Affected platforms"
+          description: "Select all platforms where the issue occurs."
+          options:
+           - label: "Android"
+           - label: "iOS"
+           - label: "HTML/GWT"
+           - label: "Windows"
+           - label: "Linux"
+           - label: "macOS"
+    - type: textarea
+      id: context
+      attributes:
+          label: Additional context
+          description: Add any other context about the problem here.
```

Comment:
Not sure, whether it's better to have less input boxes instead of more.

Reply:
Droped it